src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -1195,9 +1195,13 @@ Copyright (c) .NET Foundation. All rights reserved.
           DestinationFiles="$(SdkOutputDirectory)RuntimeIdentifierGraph.json"
           SkipUnchangedFiles="true"/>
 
+    <Copy SourceFiles="$(NuGetPackageRoot)/microsoft.netcore.platforms/$(_NETCorePlatformsPackageVersion)/PortableRuntimeIdentifierGraph.json"
+          DestinationFolder="$(SdkOutputDirectory)"
+          SkipUnchangedFiles="true"/>
+
     <GenerateSdkRuntimeIdentifierChain
           RuntimeIdentifier="$(PortableProductMonikerRid)"
-          RuntimeIdentifierGraphPath="$(SdkOutputDirectory)RuntimeIdentifierGraph.json"
+          RuntimeIdentifierGraphPath="$(SdkOutputDirectory)PortableRuntimeIdentifierGraph.json"
```

Comment:
I have absolutely no idea what that chain file is about but it should still work as expected with the portable RID graph. In general, our product build shouldn't use the legacy RID graph anymore.

Reply:
This chain file contains an ordered list of rids that are usable with the rid being built.

On the 8.0 rc2 I have installed on Fedora it contains:
```
linux-x64
linux
unix-x64
unix
any
```

I'm slightly surprised it doesn't include the source-build non-portable rid.
The CLI seems to use it for some things.